Authentication/Login/View/LoginView.swift

```diff
@@ -76,8 +93,26 @@ internal final class LoginView: UIView, LoginViewType, NibLoadable {
         }
     }
     
-    @IBOutlet weak var toSignupLabel: UILabel! {
-        didSet { toSignupLabel.text = signupLabelText }
+    // - Warning: For these buttons to be seen, this var must be
```

Comment:
Same as above, how can we know if the user set the wanted provider buttons, or if it let the default empty array because he didn't want any provider at all ?